src/Abstractions/TaskOptions.cs

```diff
@@ -100,4 +100,5 @@ public SubOrchestrationOptions(TaskOptions options, string? instanceId = null)
 /// The time when the orchestration instance should start executing. If not specified or if a date-time in the past
 /// is specified, the orchestration instance will be scheduled immediately.
 /// </param>
-public record StartOrchestrationOptions(string? InstanceId = null, DateTimeOffset? StartAt = null);
+/// <param name="ParentTraceId">Parent Trace Id.</param>
+public record StartOrchestrationOptions(string? InstanceId = null, DateTimeOffset? StartAt = null, string? ParentTraceId = null);
```

Comment:
We should not include trace information in the public API. Capturing of trace details should be done implicitly in the implementation by checking `Activity.Current`.